apps/subscriber-app-e2e/src/integration/subscription-management/subscription-management.page.ts

```diff
@@ -92,6 +92,29 @@ class SubscriptionManagementPage {
   calloutMessage() {
     return cy.xpath('//*[@role="notification"]//*[@class="message"]');
   }
+  availableChannel(subscriptionName) {
+    return cy.xpath(
+      `//tbody/tr/td[@data-testid="subscription-name" and text()="${subscriptionName}"]/following-sibling::td[2]`
+    );
+  }
+
+  availableChannelEmailIcon(subscriptionName) {
```

Comment:
The xpath is for  the email channel checked icon. It should be named emailChannelCheckedIcon(). The emailChannelIcon() should use .../div[@class=icon-0]. This is a good candidate that we ask the developer to add a data-testid as -0, -1, -2 isn't prone to change in the future.